dev-vcs/git-lfs/git-lfs-2.10.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+EGO_PN="github.com/${PN}/${PN}"
+
+if [[ ${PV} == *9999 ]]; then
+	EGIT_REPO_URI="https://${EGO_PN}"
+	inherit git-r3
+else
+	SRC_URI="https://${EGO_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~amd64-linux ~x86-linux"
+fi
+
+HOMEPAGE="https://git-lfs.github.com/"
+DESCRIPTION="command line extension and specification for managing large files with Git"
+LICENSE="Apache-2.0 BSD BSD-2 BSD-4 ISC MIT"
+
+SLOT="0"
+IUSE="doc test"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	dev-lang/go
+	doc? ( app-text/ronn )
```

Comment:
These are most likely BDEPEND with EAPI-7.

Reply:
Of course. I fixed it in my local repo.